lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -78,7 +78,7 @@ class AxeAudit extends Audit {
         node: /** @type {LH.Audit.Details.NodeValue} */ ({
           type: 'node',
           selector: Array.isArray(node.target) ? node.target.join(' ') : '',
-          path: node.path,
+          path: node.devtoolsNodePath,
```

Comment:
in connor's braindump he noted our artifacts use both `path` and `devtoolsNodePath`, and suggested we normalize to dNP.

however our public audit details type of `NodeValue` uses `path`:
https://github.com/GoogleChrome/lighthouse/blob/a1571ba4bc5b7713248ea53e01db085389e634f8/types/audit-details.d.ts#L206

this is the only property that gets renamed from artifacts to audit details... `snippet`, `selector` and even `nodeLabel` are consistent on both sides.

renaming these as `path` in the artifacts would be nice for consistency.. it does lose a bit of signal, but perhaps its worth it?  wdyt @connorjclark @patrickhulce

Reply:
I'd agree if `path` weren't so painfully ambiguous (never in a million years as a standard consumer of an LHR would I expect `path` to be this series of random integers nor would I know how to use it) 😬 

For that reason I think it's worth the cost to normalize on `devtoolsNodePath` and upgrade the renderer to support both in a future PR (which eventual removal of `.path` in some future breaking version). The fact that it also is only relevant to DevTools integration makes it easier of a change in the renderer/LHR too since it's unlikely our other big report renderer implementations (LHCI, Calibre, Treo, etc) are relying on this.

Reply:
FWIW when I was helping with adding `devtoolsNodePath` in #11061, I found it confusing when trying to remember how `NodeDetails` worked and the extra context from the name helped on the artifacts side. It would be unfortunate to lose that.

Reply:
yah sg. renaming the audit details prop to devtoolsNodePath seems like a win, even if its a bit more painful to do.